config/packages/sentry.yaml

```diff
@@ -13,15 +18,22 @@ services:
 
 when@prod:
     sentry:
-        dsn: "%sentry_dsn%"
+        dsn: '%sentry_dsn%'
         options:
             attach_stacktrace: true
             release: '%demosplan.project_name%@%project_version%'
             traces_sample_rate: "%sentry_traces_sample_rate%"
         register_error_listener: false # Disables the ErrorListener to avoid duplicate entries
+        # this hooks into critical paths of the framework (and vendors) to perform
+        # automatic instrumentation (there might be some performance penalty)
+        # https://docs.sentry.io/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation/
+        tracing:
+            enabled: false
+
 #        If you are using Monolog, you also need this additional configuration to log the errors correctly:
 #        https://docs.sentry.io/platforms/php/guides/symfony/#monolog-integration
-        register_error_handler: false
+#        register_error_listener: false
+#        register_error_handler: false
```

Comment:
I thought we're using monolog, is this for testing purposes only? But the previous configuration didn't look like it was doing much either. Maybe this line was removed simply because it is `false` by default?